src/jarabe/journal/listview.py

```diff
@@ -747,7 +747,11 @@ def __icon_clicked_cb(self, cell, path):
 
     def __cell_title_edited_cb(self, cell, path, new_text):
         iterator = self._model.get_iter(path)
-        self._model[iterator][ListModel.COLUMN_TITLE] = new_text
+        if new_text.isspace() or new_text == '':
+            return
+        else:
+            self._model[iterator][ListModel.COLUMN_TITLE] = new_text
+
         self.emit('title-edit-finished')
```

Comment:
@samdroid-apps Do you mean that if there is already existing entry it should be auto-focused on keypress?

Because according to my changes an entry cannot be renamed to empty ("") anyways. But if there is already an entry with empty name then should I make changes to search also such that it auto-focuses on empty named entries.

Is that what you are trying to suggest or there is something more to it?